developer-handbook/github-process/code-review-process.md

```diff
@@ -0,0 +1,36 @@
+# Code review
+
+You've been assign a Pull Request to review, and you want to get started.
+
+- Ensure you follow the "How to test" section, and run your own tests on top of it if you can think of other useful ones.
+
+- Keep your comments friendly; we all know we are reviewing the code and not the coder, but the lines are easily blurred (for example, try not to use words like "simply", "easily", "just", "obviously", "well, actually...").
+
+- Code review is as much a priority as writing code is, so ensure you code review promptly to unblock your colleagues: make it part of your routine. For example, you could have a look at [the PR listing page](https://github.com/pulls/review-requested) in the morning, or in between pieces of work, to see if anything needs reviewing. If you can't find time, try to set a time aside in the future and let your colleagues know when that will be.
+
+- If it's a huge PR, consider doing it in bits instead of in one go, it will be more efficient for your focus. Take your time!
+
+- Don't be afraid to ask questions! It's a prime place to learn, and it might mean that the code could be more readable, or use comments.
+
+- If a comment is not a blocker for release, make sure you state that by prefixing it with "Nit Pick".
+
+- Don't be a perfectionist, but keep an eye out for:
+	- Our own chosen standards (naming conventions, file structure, etc.)
+	- Readability
+	- Compliance (e.g. a11y)
+	- Dead code
+	- Performance concerns
+	- Security concerns
+	- Gaps in logic (e.g. in an `if` statement, is every case covered?)
+	- Duplication of code (e.g. helpers that we already have a library for)
+	- Edge cases
+	- Leftovers from development, (e.g. `console.log`, addressed `TODO`s)
+	- Typos!
```

Comment:
Do the tests cover all the cases?

Reply:
Some thoughts on writing/reviewing tests (probably too much for this section, but could be boiled down a bit)
* Do they adequately reflect the questions asked in the ticket?
* Do they cover appropriate boundary conditions? (a common shortcoming is that tests tend to cover what you want to happen, but skip over what happens when the input is "wrong")
* Do the tests tell a story? Are they easily readable?
* Do they actually test what they say they are testing?

Reply:
I'm adding this in the list above
> - Do the tests cover all cases? Do they actually test what they say they are testing?

But I liked the other questions for the "Pull request process" as they're a good indicator of what tests should be like! So adding it as a list here https://github.com/wellcomecollection/developer-handbook/pull/1/files#diff-98802f63a1e1ae0bb73422d2c806ebd85275584be6ecf804d2b03e002481b74dR34

> - Would new tests be relevant? If so, add them:
>    - Do they adequately reflect the questions asked in the ticket?
>    - Do they cover appropriate boundary conditions? (tests tend to cover what you want to happen, but should also cover what happens when the input is "wrong")
>    - Do the tests tell a story? Are they easily readable?